### PR TITLE
Badminton Product

### PR DESCRIPTION
**Pull request by utkarshducoder**
please review my badminton page

live link: https://sportssite.netlify.app/html/badminton.html

this page contains following features:

- nav: contains brand name and link for the home page.
- sidebar: contains links to all the filters.
- div: contains the main documentation of page.

f3

### CODE REVIEW

File: badminton/badminton.css [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+.centered {
+  position: absolute;
+  top: 85%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: large;
+  font-weight: bold;
+  font-size: 80px;
+  color: white;
```
Comment (khushijohri2001) [inline badminton/badminton.css]: use var here

File: badminton/badminton.css [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+}
+.col-4 img{
+    width: 100%;
+}
+.featured-heading{
+    text-align: center;
+    margin: 0 auto 80px;
+    position: relative;
+    line-height: 60px;
+    color: #555;
```
Comment (khushijohri2001) [inline badminton/badminton.css]: use var here

File: badminton/badminton.css [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+}
+
+nav{
+background: #0082e6;
```
Comment (khushijohri2001) [inline badminton/badminton.css]: use var here

**Review by khushijohri2001 — APPROVED** [review]

File: badminton/badminton.css [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+display: none;
+}
+#check{
+display: none;
+}
+/**/
+@media (max-width: 480px){
+label.logo{
+  font-size: 30px;
+  padding-left: 50px;
```
Comment (i-am-a-codegirl) [inline badminton/badminton.css]: shift media queries in the end

**Review by i-am-a-codegirl — APPROVED** [review]

**Pull request merged**